appveyor.yml

```diff
@@ -1,64 +0,0 @@
-
```

Comment:
I guess it doesn't hurt to add `python` to `requirements/build` and commit, right?